python/py-sentence-transformers/Portfile

```diff
@@ -40,9 +40,9 @@ homepage            https://github.com/UKPLab/sentence-transformers
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}/
 distname            ${python.rootname}-${version}
```

Comment:
The fields for ‘master_sites’ and ‘distname’ are set now automatically by the python PG. please remove these two lines

Reply:
Done thanks.